lib/fathead/spring-framework/fetch.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+# see http://stackoverflow.com/a/10959815/834
+rm -rf download
+rm -rf docs
+mkdir -p download
+mkdir -p docs/api
+cd download
+unamestr=`uname`
+if [[ "$unamestr" == 'Darwin' ]]; then
+   curl -O http://docs.spring.io/spring/docs/current/javadoc-api/allclasses-noframe.html
+else
+  wget --quiet --no-check-certificate --no-cookies -O http://docs.spring.io/spring/docs/current/javadoc-api/allclasses-noframe.html
```

Comment:
Looks like you have the `-O` in the wrong place? Right now the download URL isn't being specified